File: app/src/main/java/com/example/myittaroostockinventorymanger/local/Stock.java
package com.example.myittaroostockinventorymanger.local;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Stock {

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "stock_id")
    private long stockId;
    private String name;

    public Stock(String name) {
        this.name = name;
    }

    public long getStockId() {
        return stockId;
    }

    public void setStockId(long stockId) {
        this.stockId = stockId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Stock{" +
                "stockId=" + stockId +
                ", name='" + name + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/myittaroostockinventorymanger/local/Stock.kt
package com.example.myittaroostockinventorymanger.local

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Parcelize
@Entity
data class Stock(
        var name: String
) : Parcelable {
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "stock_id")
     var stockId: Int =0
}